src/features/harborView/forms/harborForm/HarborEditForm.tsx

```diff
@@ -17,7 +16,7 @@ import { getHarbor, mapValuesToMutation } from './utils/utils';
 import { UPDATE_HARBOR_MUTATION } from './mutations';
 import { UPDATE_HARBOR, UPDATE_HARBORVariables as UPDATE_HARBOR_VARS } from './__generated__/UPDATE_HARBOR';
 import { HARBOR_FORM } from './__generated__/HARBOR_FORM';
-import FileList, { PersistedFile } from '../../../../common/fileList/FileList';
+import { PersistedFile } from '../../../../common/fileList/FileList';
```

Comment:
This import should be removed too, as well as the validation schema related to this file upload

Reply:
Good catch, will whack them.

Reply:
done.